ci.mk

```diff
@@ -215,7 +219,7 @@ $(misra_suppresions): $(misra_cppcheck_supressions) $(misra_deviation_suppressio
 	@cat $^ > $@
 
 misra-check: $(misra_rules) $(cppcheck_type_cfg) $(misra_suppresions)
-	@$(CPPCHECK) $(cppcheck_misra_flags) $(_misra_c_files)
+	@$(CPPCHECK) $(cppcheck_misra_flags) $(_misra_flags) $(_misra_c_files) $(_misra_sup)
```

Comment:
Can't we include `_misra_flags` in `cppcheck_misra_flags`?

Reply:
No. I can't totally picture why, but if I include the `_misra_flags` in `cppcheck_misra_flags`, the variable is not expanded on the time of the call. If you have any suggestion I can try to introduce it, but for now I can't figure out a solution.

Reply:
So why did you change it? The last commit does exactly this.

Reply:
By mistake. I just amend it.

Reply:
I think we just need to make `cppcheck_misra_flags` recursively expanded. That is, changing `cppcheck_misra_flags:=...` to `cppcheck_misra_flags=...`

Reply:
That did the work. Just changed :)